cli.py

```diff
@@ -51,38 +52,46 @@ def attr_upd():
         return
 
     backend_selector = BackendSelector(selected_brand, selected_region)
+    session = aiohttp.ClientSession()
 
-    auth = Auth(backend_selector, args.email, args.password)
+    auth = Auth(backend_selector, args.email, args.password, session)
     await auth.do_auth(store=False)
-    appliance_manager = AppliancesManager(backend_selector, auth)
+    appliance_manager = AppliancesManager(backend_selector, auth=auth, session=session)
     if not await appliance_manager.fetch_appliances():
         logger.error("Could not fetch appliances")
+        await session.close()
```

Comment:
instead of having to keep track of where we close the session, could we instead use `with` like:
`async with aiohttp.ClientSession() as session:` and just put all the code inside that block?